Avatar.html

```diff
@@ -141,7 +144,7 @@ <h3 id="Badgegalaxy">  </h3>
 					<div class="col-xs-4">
 						<div class="container-outer">
 							<div class="container-inner hair">
-								<img src="img/avatar/Hair_01.png" " id="001">
```

Comment:
I keep getting an error message of "Special characters must be escaped" 

<img width="1041" alt="screen shot 2017-09-30 at 6 10 43 pm" src="https://user-images.githubusercontent.com/13709857/31050845-6e129634-a60c-11e7-85b6-33f7905c4268.png">

Reply:
You have a " in front of id="001" - if you remove it you should be fine